### PR TITLE
version check precedence

### PR DESCRIPTION
**Pull request by rotem-codefresh**

### CODE REVIEW

File: pkg/reporter/reporter.go [hunk 1]
```diff
@@ -75,7 +75,6 @@ const (
 	InstallStepRunPreCheckEnsureCliVersion            CliStep = "install.run.pre-check.step.ensure-cli-version"
 	InstallStepRunPreCheckRuntimeCollision            CliStep = "install.run.pre-check.step.runtime-collision"
 	InstallStepRunPreCheckExisitingRuntimes           CliStep = "install.run.pre-check.step.existing-runtimes"
-	InstallStepRunPreCheckClusterChecks               CliStep = "install.run.pre-check.step.cluster-checks"
```
Comment (roi-codefresh) [inline pkg/reporter/reporter.go:78]: @danielm-codefresh

File: pkg/util/kube/kube.go [hunk 2]
```diff
@@ -167,6 +173,155 @@ func EnsureClusterRequirements(ctx context.Context, kubeFactory kube.Factory, na
 		return fmt.Errorf("%s: %v", requirementsValidationErrorMessage, specificErrorMessages)
 	}
 
+	err = runNetworkTest(ctx, kubeFactory, currentContext.URL)
```
Comment (roi-codefresh) [inline pkg/util/kube/kube.go]: just pass in the url if you don't use anything else from `config.AuthContext`

**Review by roi-codefresh — APPROVED** [review]

File: pkg/reporter/reporter.go (hunk 1 above)
Comment (danielm-codefresh) [reply pkg/reporter/reporter.go:78]: Why did you remove this step?

File: pkg/util/kube/kube.go (hunk 2 above)
Comment (danielm-codefresh) [reply pkg/util/kube/kube.go]: Please add reporting here

**Review by roi-codefresh — APPROVED** [review]

**Pull request merged**